.github/workflows/steam.yml

```diff
@@ -2,6 +2,8 @@ name: Steam depot
 
 on:
   workflow_dispatch:
+  pull_request:
+    types: [synchronize]
```

Comment:
```suggestion
```

Reply:
As soon as it's confirmed to be fixed :)